### PR TITLE
Added brightness and scenes functionality for Vera lights

### PR DESCRIPTION
**Pull request by NickBorgers**
The changes are actually much less substantial than git is showing. I had some formatting problems going from my Windows environment to my Linux server and got a bit overzealous with my formatting corrections. In particular, I replaced all tabs with 4 spaces.

But yeah, reasonably self-explanatory upgrades. I used all of your style as relevant.

I also added a lot to the README because I had a very hard time getting started. I had never used Flask, or whatever it is that turns Python into a webserver. I tried to hook up it in CGI style... and yeah that took a while screwing around to figure out why it would the web request thread would run forever.

I am using an HTML/JS web interface but as mentioned in the README, don't really want to share it because I license it to a company I own and that would get all wonky. If you do circle back to this project AND work on a generic web interface instead of an Android app I would be willing to contribute a bit.

### CODE REVIEW

File: device.py [hunk 1]
```diff
@@ -10,18 +11,28 @@ class Device():
...
+    brightness = 0
+    hasBrightness = False
 
-    def __init__(self, id, name, room, state):
+    def __init__(self, id, name, room, state, hasBrightness, brightness):
         self.id = id
         self.name = name
         self.room = room
         self.state = state
+        self.hasBrightness = hasBrightness
```
Comment (bencxr) [inline device.py]: hasBrightness seems to be redundant. We can check if brightness was None.

**Comment by christinang89** [comment]
Thanks for working on this @ug2215 ! If you can make the change above, I can go ahead and merge it :)

File: device.py (hunk 1 above)
Comment (NickBorgers) [inline device.py]: Sure; I've been writing a lot of Java lately so I've gotten my mind out of the duck-typing game. I will make the adjustment.

**Comment by christinang89** [comment]
thanks for doing this! upon closer inspection, i probably would send the brightness info in the lights endpoint together with the state since this design isnt exactly very RESTful, but maybe i'll revisit this again later :)

**Pull request merged**